docs_outdoornav_user_manual/release_notes.mdx

```diff
@@ -6,6 +6,26 @@ toc_min_heading_level: 2
 toc_max_heading_level: 4
 ---
 
+## 0.9.0
+
+### New Features
+
+-   Operators can now Schedule Missions to run at specific times
+-   Added in support for BulletCat12 Microhard WiFi and Cellular connections
+-   Allow Audio recording as both tasks and manual operations if UGV has Microphones
+-   If installed with PDU, UGV can be set to Low Power mode to better conserve power
+
+### Bug Fixes 
+
+-   1340: Undocking not working through tasks
+
+### Known Issues 
+
+-   131: Software upgrade process not documented fully.
+-   609: Realsense D435 collision avoidance not fully tuned.
+-   751: Wireless icon in UI doesn't work for Ubiquity hardware.
+-   1324: Allow single Waypoint missions and/or tasks on first waypoint (required for undocking through tasks).

```

Comment:
This is marked as "Done" in Jira. Can we remove it from the list of known issues?